app/controllers/admin/navigation_links_controller.rb

```diff
@@ -14,6 +14,7 @@ def index
     def create
       navigation_link = NavigationLink.new(navigation_link_params)
       if navigation_link.save
+        Rails.cache.delete("navigation_links")
```

Comment:
We want to delete the cache every time theres a CRUD action on navigation_links

Reply:
non-blocking suggestion: seems like it is part of `after_action` and I see there's something similar there already.